queries/supercollider/highlights.scm

```diff
@@ -0,0 +1,109 @@
+; highlights.scm
+; See this for full list: https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md
+
+; comments
+(line_comment) @comment
+(block_comment) @comment
+
+; Argument definition
+
+; Fixme: Should be attribute but nvim-treesitter doesn't supprt it at the moment
+(argument name: (identifier) @parameter) 
+;(argument name: (identifier) @parameter)  
```

Comment:
I think there was a discussion of using `property` instead of attribute.

Plu a minor duplication : 
```suggestion
; Fixme: Should be attribute but nvim-treesitter doesn't supprt it at the moment
(argument name: (identifier) @parameter) 
```